README.md

```diff
@@ -21,6 +21,34 @@ CPython](https://github.com/python/cpython/blob/e0433c1e7/Doc/tools/rstlint.py).
 - focus on finding errors that are **not** visible to sphinx-build.
 
 
+## Using sphinx-lint
+
+To use sphinx-lint, simply run:
+
+```console
+$ sphinx-lint file_or_directory_1, file_or_directory_2, ...
+```
```

Comment:
If there's no output to show, this makes it easier to copy/paste without the dollar:

```suggestion
```sh
sphinx-lint file_or_directory_1, file_or_directory_2, ...
```
```